custom_components/tahoma/__init__.py

```diff
@@ -174,6 +175,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         if device.widget == HOMEKIT_STACK:
             print_homekit_setup_code(device)
 
+    supported_platforms = set(platforms.keys())
+    # Required to load the low speed switch
+    supported_platforms.add(SWITCH)
```

Comment:
@iMicknl At the end we will have SWITCH, SENSOR, BINARY SENSOR. I guess we should add all the other ones (not in this PR). What do you think?